doc/fileformat/asc.rst

```diff
@@ -0,0 +1,58 @@
+.. _formatasc:
+
+Neurlucida ASCII format
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Arbor has support for reading cable cell morphologies described using the
+`Neurlucida ASCII file format <https://www.mbfbioscience.com/help/pdf/NL9.pdf>`_
+or ``.asc`` file format.
+
+Because ASCII files contain both a morphology description and meta data, the
+loader returns both a morphology, and a label dictionary that describes regions
+and locsets from the meta data.
+
+.. warning::
+    The ASCII file format has no formal specification, and describes not only the cell
+    morphology and features like spines, but rich meta data about markers and colors.
+    Not all of the information is relevant for simulation.
+
+    Arbor's ASCII importer discards descriptions that it determines are
+    not relevant for simulation (e.g. color meta-data attached to a dendrite).
+    However Arbor will throw an error when it encounters content that it can't interpret,
+    unlike some other readers that will ignore anything they don't recognise.
```

Comment:
The only meta-data labelled in the dictionary in this implementation are the standard regions of soma, axon, dend and apic. These are also tagged with the same tag numbers used by SWC.

It would certainly be possible to add locset information specific to asc files, like spine locations and terminal types. However, I didn't do it here because, it isn't trivial and NEURON doesn't do it, so it isn't required for a minimal working implementation.

Further, if we want to modify the morphology once it is loaded, e.g. editing the axon, this becomes more challenging, and it could implement an approach that works for all imported file types (NeuroML, SWC, asc, etc)

Reply:
Ok, sad to loose that information, but it's really not relevant to simulation. Although placing things at these special locations might be interesting to some folks, it should wait for a user request.